bucket/ghostscript.json

```diff
@@ -1,19 +1,19 @@
 {
     "homepage": "https://www.ghostscript.com",
-    "version": "9.26",
+    "version": "9.27",
     "license": "AGPL-3.0",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs926/gs926aw64.exe#/dl.7z",
-            "hash": "sha512:4c961b852bd207e9a1cae00355c5e582ad047dec54912ae5e7d326c4eacc0c8300bf49164d7ba5817f9ab5a7b9cb5d60f235b162dd30c0d6e8706566f7e0a605",
+            "url": "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs927/gs927w64.exe#/dl.7z",
+            "hash": "sha512:49fc973bfae12f366a837bcc39d93e3d8e3c88359f7d45279df899f0b2941c264efc47815966867d332b32e87ca8d88697edf7792b162645ed7a549cecc373b2",
             "bin": [
                 "bin\\gswin64.exe",
                 "bin\\gswin64c.exe"
             ]
         },
         "32bit": {
-            "url": "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs926/gs926aw32.exe#/dl.7z",
-            "hash": "sha512:5e3f9e8303e1b65c9c6b67dbd3c5a999c7b7eeafe499638d3bbe718ccb1a1b26928dd441d65431b423f385b1a70d698e5027f7ee701c963eb410d3f79fde5a40",
+            "url": "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs927/gs927w32.exe#/dl.7z",
+            "hash": "sha512:ef18548c62a5107f951d839d7c83ea5e03b35b2b85fd2c820b95687fb28e9ebfc799c2efe87d6db16879a8d9d995fe56f0b6a1248df0d6ee41d5851edfce9f83",
             "bin": [
```

Comment:
Why to create separate bin, when bin folder is added to path??

Reply:
No `bin` dir should ok.